e2e/commands/start/start_test.go

```diff
@@ -96,9 +99,16 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 		})
 	})
 
+	When("--project-path is passed", func() {
+		It("Checks if the project path property was set", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(`\"disable_docker\": true`))
+		})
+	})
+
 	When("--disable-docker is passed", func() {
 		It("Checks if the disable docker was set as true", func() {
-			Expect(session.Out.Contents()).To(ContainSubstring(`\"disable_docker\": true`))
+			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`\"project_path\": \"%s\"`, testutil.NormalizePathToAssertInJSON(projectPath))))
+			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`Project sent to folder in location: [%s`, testutil.NormalizePathToAssert(projectPath))))
```

Comment:
seems like you misplaced the tests, this should be on `When("--project-path is passed", func()` and the other in `When("--disable-docker is passed", func() {`

Reply:
Fixed!